src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -260,6 +260,11 @@ protected override async Task StartAsyncCore(CancellationToken cancellationToken
         public async Task InitializeAsync()
         {
             _stopwatch.Start();
+            if (!_environment.IsPlaceholderModeEnabled())
+            {
+                string runtimeLanguage = string.IsNullOrEmpty(_currentRuntimelanguage) ? "none" : _currentRuntimelanguage;
```

Comment:
nit: can you fix the casing of _currentRuntimelanguage? I realize you didn't add that member :)